### PR TITLE
Add FundraiseUp script to `<head>`

### PR DESCRIPTION
**Pull request by shawninder**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -4,10 +4,27 @@
  * Additional code for the child theme goes in here.
  */
 
-add_action( 'wp_enqueue_scripts', 'enqueue_child_styles', 99);
+add_action('wp_enqueue_scripts', 'enqueue_child_styles', 99);
+add_action('wp_head', 'hook_javascript');
 
-function enqueue_child_styles() {
+function enqueue_child_styles()
+{
```
Comment (techieshark) [inline functions.php]: This really is just stylistic, but I think the [PHP code style guide](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#brace-style) suggests to put brackets on the same line, like this -

```suggestion
function enqueue_child_styles() {
```

File: functions.php [hunk 2]
```diff
@@ -4,10 +4,27 @@
...
 	$css_creation = filectime(get_stylesheet_directory() . '/style.css');
 
-	wp_enqueue_style( 'child-style', get_stylesheet_directory_uri() . '/style.css', [], $css_creation );
+	wp_enqueue_style('child-style', get_stylesheet_directory_uri() . '/style.css', [], $css_creation);
+}
+
+function hook_javascript()
+{
```
Comment (techieshark) [inline functions.php]: Again, purely stylistic! Based on my understanding of the PHP style guide -

```suggestion
function hook_javascript() {
```

File: functions.php [hunk 3]
```diff
@@ -4,10 +4,27 @@
...
+?>
+<!-- Fundraise Up: world-class checkout experience for serious online fundraising -->
```
Comment (techieshark) [inline functions.php]: Unless we're getting paid to advertise, we can save our supporters a few bytes of bandwidth 😄 

```suggestion
<!-- Begin Fundraise Up -->
```

**Review by techieshark — COMMENTED** [review]
Hi @shawninder  - Peter (from GPAP) here - 

I'm excited to hear how you go with this! 

I'm no PHP expert but I hope a few little suggestions is helpful. Cheers!

~ Peter

**Comment by shawninder** [comment]
> Hi @shawninder - Peter (from GPAP) here -
> 
> I'm excited to hear how you go with this!
> 
> I'm no PHP expert but I hope a few little suggestions is helpful. Cheers!
> 
> ~ Peter

Thanks Peter!

I guess I got used to minification solving all of these things for me in more optimized tech stacks.

File: functions.php [hunk 4]
```diff
@@ -1,13 +1,23 @@
...
+add_action('wp_enqueue_scripts', 'enqueue_child_styles', 99);
+add_action('wp_head', 'hook_javascript');
 
 function enqueue_child_styles() {
 	$css_creation = filectime(get_stylesheet_directory() . '/style.css');
+	wp_enqueue_style('child-style', get_stylesheet_directory_uri() . '/style.css', [], $css_creation);
+}
 
-	wp_enqueue_style( 'child-style', get_stylesheet_directory_uri() . '/style.css', [], $css_creation );
+function hook_javascript() {
```
Comment (lithrel) [inline functions.php]: Tested locally, it works :+1: 
I would rename this function to something more explicit like `include_fundraiseup_script`, but that works anyway :)

Comment (shawninder) [reply functions.php]: Good idea, thanks

**Comment by shawninder** [comment]
Can someone merge this? What is the protocol for getting this into production?

**Comment by techieshark** [comment]
Good question @shawninder - I'm actually not sure. I see this:

image

so clearly I don't have access. Maybe @lithrel could shed some light?

**Pull request merged**